src/ui/networkpage.rs

```diff
@@ -68,18 +68,24 @@ fn info_row_from_iface<'a, 'b>(iface: &'a NetworkInterfaceStatus) -> Row<'b> {
 
     let height = (ipv4_len + ipv6_len).max(1);
 
-    // join both ipv4 and ipv6 addresses and separate by newline
+    // join Ipv4 and Ipv6 addresses and separate by newline
     let combined_ip_list_iter = iface
         .ipv4
         .iter()
-        .chain(iface.ipv6.iter())
-        .flat_map(|v| v.iter().cloned())
+        .flat_map(|v| v.iter())
         .map(|ip| ip.to_string())
+        .chain(
+            iface
+                .ipv6
+                .iter()
+                .flat_map(|v| v.iter())
+                .map(|ip| ip.to_string()),
+        )
         .collect::<Vec<_>>()
         .join("\n");
 
     // cell #3 IP address list
-    if height > 1 {
+    if height > 0 {
```

Comment:
I'm not that good with the code base (yet :D) why 0 instead of 1?

Reply:
this is a real bug here: if we have a single IP (height 1) we display ... nothing :)